Xamarin.Forms.Core/Span.cs

```diff
@@ -67,6 +67,8 @@ public string Text
 
 		public static readonly BindableProperty FontAttributesProperty = FontElement.FontAttributesProperty;
 
+		public static readonly BindableProperty LineHeightProperty = Label.LineHeightProperty;
```

Comment:
see previous comment, should be something like `LineHeightElement.LineHeightProperty`

Reply:
Should we add it to `ITextElement` and `TextElement` or create `ILineHeightElement/LineHeightElement`? Noticed you mentioned TextElement in the issue description.